frontend/src/components/common/dokumente/Dokumente.vue

```diff
@@ -65,6 +72,8 @@ export default class Dokumente extends Mixins(
 
   private allowedMimeTypes = "";
 
+  private loading = false;
```

Comment:
Das Attribut `loading` in der Componente `v-progress-circular` nicht direkt referenzieren sondern einen Vue Getter verwenden.

z.B.

```ts
get isLoading() {
 return this.leading;
}
```

Reply:
erledigt